assets/training/distillation/src/common/constants.py

```diff
@@ -53,7 +53,7 @@
 DEFAULT_SUCCESS_RATIO = 0.7
 DEFAULT_REQUEST_BATCH_SIZE = 10
 MAX_BATCH_SIZE = 100
-MIN_RECORDS_FOR_FT = 65
+MIN_RECORDS_FOR_FT = 40
```

Comment:
Could we please verify this for other tasks that are already released?

Reply:
Reverted back to original and allowed the min records to be lower for math tasks as that has been tested already